webapp/src/components/backstage/playbook_editor/outline/inputs/retrospective_interval_selector.tsx

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {useIntl} from 'react-intl';
+
+import React, {useMemo} from 'react';
+import {Duration} from 'luxon';
+
+import {Mode, Option, useMakeOption} from 'src/components/datetime_input';
+import {StyledSelect} from 'src/components/backstage/styles';
+
+interface Props {
+    seconds: number;
+    setSeconds: (seconds: number) => void;
```

Comment:
nit: onChange would be more accurate
```suggestion
    onChange: (seconds: number) => void;
```